CefGlue.Packages.props

```
@@ -3,10 +3,12 @@
     <ItemGroup Condition="'$(Platform)' == 'x64'">
         <PackageReference PrivateAssets="None" Include="chromiumembeddedframework.runtime.win-x64" />
         <PackageReference PrivateAssets="None" Include="cef.redist.osx64" />
+        <PackageReference PrivateAssets="None" Include="cef.bin.linux64" />
```

this package will need to be updated when we update CefGlue, since we dont have control over it, it can be problematic for us to evolve 
```suggestion
        <PackageReference PrivateAssets="None" Include="cef.bin.linux64" />
```

---

Yes I know that problem. I want to transfer those packages to OutSystems' NuGet and GitHub account. I can do the GitHub one easily but I have no idea how to do the NuGet packages.

---

Sent a request on NuGet. Sending request on GitHub gave me error "You don’t have the permission to create public repositories on OutSystems".